includes/class-edd-customer.php

```diff
@@ -699,6 +703,22 @@ public function decrease_value( $value = 0.00 ) {
 		return $this->purchase_value;
 	}
 
+	/**
+	 * Gets the additional emails of a user
+	 *
+	 * @since  x.y.z
+	 * @return array           The emails of the customer.
+	 */
+	public function get_emails() {
+		if ( array_key_exists( 'emails', $this->data  ) ) {
+            		return $this->data['emails'];
+        	} else {
+			$this->data['emails']   = (array) $this->get_meta( 'additional_email', false );
+			$this->data['emails'][] = $this->email;
```

Comment:
While we're in here, one thing that's always annoyed me is that the primary email is the `last` email in the array...can we make it so `$this->email` is the first value in the array of emails returned? It makes more sense for that to be first than last.

Reply:
@cklosowski It's been updated to support this

Reply:
👍